marketplace/core/types/externals/omie/views.py

```diff
@@ -27,3 +30,11 @@ def configure(self, request, **kwargs):
         self.perform_update(serializer)
 
         return Response(serializer.data)
+
+    def perform_destroy(self, instance):
+        channel_uuid = instance.config.get("channelUuid")
+        if channel_uuid:
+            client = FlowsClient()
+            client.release_external_service(channel_uuid, self.request.user.email)
+
+        super().perform_destroy(instance)
```

Comment:
Your class inherits methods from the `BaseAppTypeViewSet`, in this section the `super().perform_destroy(instance)` you are triggering the existing performe_destroy on `BaseAppTypeViewSet`, and performing another delete, which is designed for channels. 

To resolve this, you can rewrite the native method of the `DestroyModelMixin` class and add your code snippet in this method.

`    
  def destroy(self, request, *args, **kwargs):
        instance = self.get_object()
        self.perform_destroy(instance)
        return Response(status=status.HTTP_204_NO_CONTENT)

   def perform_destroy(self, instance):
        instance.delete()
`